### PR TITLE
Added new recipe Irish winter spiked coffee

### PR DESCRIPTION
**Pull request by divyhshah**

### CODE REVIEW

File: src/recipes/irish-winter.json [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    
+    ],
+    "directions": [
+        "Gather the ingredients.",
+        "In a warm mug or Irish coffee glass, pour the Irish whiskey, coffee liqueur, and Irish cream.",
+        "Fill with hot coffee.",
+        "Stir to combine",
+        "Top with whipped cream."
+    ],
+    "image": "IrishWinter.jpg",
```
Comment (alfg) [inline src/recipes/irish-winter.json]: Can we make this all lowercase and hyphens?
```suggestion
    "image": "irish-winter.jpg",
```

Comment (alfg) [reply src/recipes/irish-winter.json]: Also, remove the old image.

Comment (divyhshah) [reply src/recipes/irish-winter.json]: i already removed @alfg

Comment (alfg) [reply src/recipes/irish-winter.json]: I still can see `src/recipes/img/IrishWinter.jpg` in this PR>

File: src/recipes/irish-winter.json [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+{
+            "quantity": "3/4",
+            "measure": "ounces",
+            "ingredient": "Irish cream liqueur"
+        },
+        {
+            "quantity": "4",
+            "measure": "ounces",
+            "ingredient": "coffee"
+        },
```
Comment (alfg) [inline src/recipes/irish-winter.json]:
```suggestion
        }
```

Comment (divyhshah) [reply src/recipes/irish-winter.json]: @alfg done !!

**Comment by alfg** [comment]
Thanks!

**Comment by divyhshah** [comment]
@alfg  i am not able to see my recipe on website.

**Comment by alfg** [comment]
Should be there now:
https://opendrinks.netlify.com/recipe/irish-winter

**Pull request merged**